contracts/GateSeal.vy

```diff
@@ -74,6 +74,11 @@ EXPIRY_MUST_BE_FUTURE: constant(String[39]) = "expiry timestamp: must be in the
 EXPIRY_EXCEEDS_MAX: constant(String[43]) = "expiry timestamp: exceeds max expiry period"
 SEALABLES_INCLUDES_ZERO: constant(String[32]) = "sealables: includes zero address"
 SEALABLES_INCLUDES_DUPLICATES: constant(String[30]) = "sealables: includes duplicates"
+DAO_ZERO_ADDRESS: constant(String[18]) = "dao: zero address"
+EXTENSION_DURATION_ZERO: constant(String[24]) = "extension duration: zero"
+SENDER_NOT_DAO: constant(String[17]) = "sender: not DAO"
+EXTENSIONS_EXHAUSTED: constant(String[22]) = "extensions: exhausted"
+GATE_SEAL_EXPIRED: constant(String[18]) = "gate seal: expired"
 
```

Comment:
I think it's better to inline the errors, so errors are readable and accessible at the precise place they are thrown